releasenotes/notes/sbm-random-graph-bf7ccd8e938f4218.yaml

```diff
@@ -0,0 +1,9 @@
+features:
+  - |
+    Adds new random graph generator in rustworkx for the stochastic block model.
+    There is a generator for directed :func:`.directed_sbm_random_graph` and
+    undirected graphs :func:`.undirected_sbm_random_graph`.
+  - |
+    Adds new function ``sbm_random_graph`` to the rustworkx-core module
+    ``rustworkx_core::generators`` that samples a graph from the stochastic
+    block model.
```

Comment:
You can add drawings of some example graphs if you want, check https://github.com/Qiskit/rustworkx/blob/955d9890ac7073c3eb8878e4fdcfeedfeef3e73d/releasenotes/notes/0.14/add-albert-graph-5a7d393e1fe18e9d.yaml#L5 for an example